.github/ISSUE_TEMPLATE/10-bug-report.yaml

```diff
@@ -0,0 +1,105 @@
+name: Bug Report
+description: If you have a reproducible in-game issue
+labels: ["Bug"]
+assignees: Nightenom
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+        - label: I have searched the existing issues.
+          required: true
+  - type: checkboxes
+    attributes:
+      label: Are you using the latest Structurize Version?
```

Comment:
Should this be the "latest Structurize" version or the "Structurize version for your version of MineColonies"?

Reply:
issues here are for standalone structurize bugs, if you cant reproduce the issue without mcol, then it's probably mcol issue